src/shell/lean.cpp

```diff
@@ -491,9 +489,6 @@ int main(int argc, char ** argv) {
         case 'n':
             native_output         = optarg;
             break;
-        case 'r':
-            doc = optarg;
```

Comment:
Could you also remove the `doc` variable and the `leandoc.cpp` file, please?

Reply:
Whoops, knew I'd miss something... Fixed.